src/Migrate/custom/New-AzMigrateDiskMapping.ps1

```diff
@@ -51,7 +51,25 @@ function New-AzMigrateDiskMapping {
     )
     
     process {
-    
+        # Validate the inputs
```

Comment:
please use the following style of “if else"
```powershell
if (something) {
}
elseif (something) 
```
Please take care of the space between "if" and "(" or ")" and "{".

Reply:
You can use validate set to validate DiskType

Reply:
I have removed the if-else blocks, and used ValidateSet.